test/test_documents_api.py

```diff
@@ -35,57 +35,144 @@ def test_assign_document(self):
 
         Assign a Document  # noqa: E501
         """
-        pass
+        mock_client = Mock()
+        mock_client.call_api.return_value = {"id": 46530}
+        doc_api = DocumentsApi(mock_client)
+
+        body = {
+            "id": 46530,
+            "email": "user@email.com",
+            "is_translator": True,
+            "is_reviewer": False,
+            "due_date": "2019-10-16T22:12:34.000Z",
+        }
+
+        assert doc_api.assign_document(body) == {"id": 46530}
```

Comment:
On second thought I think we may want to mock out a different method for these types of tests. The return value of these api methods should be an object with a specific type, not a dict. It looks like that `call_api` method handles the deserialization as well so we shouldn't mock that. There is actually a `request` method [here](https://github.com/lilt/lilt-python/blob/master/lilt/api_client.py#L369) that just handles the http call.

I think we could update all of these to something like

```python
mock_client = Mock()
mock_response = Mock()
mock_response.data = json.dumps({"id": 46530})
mock_client.request.return_value = mock_response
doc_api = DocumentsApi(mock_client)

body = {
    "id": 46530,
    "email": "user@email.com",
    "is_translator": True,
    "is_reviewer": False,
    "due_date": "2019-10-16T22:12:34.000Z",
}

assignment_response = doc_api.assign_document(body)
assert assignment_response.id == 46530
```

I think the main goal of these tests should be to test the deserialization from the json api response to the appropriate data type.